packages/inter-protocol/test/swingsetTests/psmUpgrade/bootstrap-psm-upgrade.js

```diff
@@ -0,0 +1,313 @@
+  let psmPublicFacet;
+
+  /** @type {Subscriber<import('../../../src/psm/psm.js').MetricsNotification>} */
+  let metrics;
+  /** @type {UpdateRecord<import('../../../src/psm/psm.js').MetricsNotification>} */
+  let metricsRecord;
+
+  /** @type {VatAdminSvc} */
+  let vatAdmin;
+
+  let initialPoserInvitation;
+  let poserInvitationAmount;
+
+  // for startInstance
+  /**
+   * @type {{
+   * committee?: Installation<import('@agoric/governance/src/committee').start>,
+   * psmV1?: Installation<PsmSF>,
+   * puppetContractGovernor?: Installation<import('@agoric/governance/tools/puppetContractGovernor').start>,
+   * }}
+   */
+  const installations = {};
+
+  /** @type {import('@agoric/governance/tools/puppetContractGovernor').PuppetContractGovernorKit<PsmSF>} */
+  let governorFacets;
+
+  /** @type {Omit<import('@agoric/zoe/src/zoeService/utils.js').StartParams<PsmSF>['terms'], 'issuers' | 'brands'>} */
+  const psmTerms = {
+    anchorBrand: anchor.brand,
+
+    electionManager: Far('mock instance handle'),
+    governedParams: {
+      // @ts-expect-error missing value, to be filled in
+      [CONTRACT_ELECTORATE]: {
+        type: ParamTypes.INVITATION,
+      },
+    },
+  };
+  const staticPrivateArgs = {
+    storageNode: storageKit.rootNode,
+    marshaller,
+  };
+
+  /**
+   * @param {Amount<'nat'>} giveAnchor
+   * @param {Amount<'nat'>} [wantMinted]
+   */
+  const swapAnchorForMintedSeat = async (giveAnchor, wantMinted) => {
+    const seat = E(zoeService).offer(
+      E(psmPublicFacet).makeWantMintedInvitation(),
+      harden({
+        give: { In: giveAnchor },
+        ...(wantMinted ? { want: { Out: wantMinted } } : {}),
+      }),
+      harden({ In: anchor.mint.mintPayment(giveAnchor) }),
+    );
+    return seat;
+  };
+
+  return Far('root', {
+    /**
+     *
+     * @param {{
+     * vatAdmin: ReturnType<import('@agoric/swingset-vat/src/vats/vat-admin/vat-vat-admin')['buildRootObject']>,
+     * zoe: ReturnType<import('@agoric/vats/src/vat-zoe')['buildRootObject']>,
+     * }} vats
+     * @param {*} devices
+     */
+    bootstrap: async (vats, devices) => {
+      vatAdmin = await E(vats.vatAdmin).createVatAdminService(devices.vatAdmin);
+      ({ feeMintAccess, zoeService } = await E(vats.zoe).buildZoe(
+        vatAdmin,
+        undefined,
+        'zcf',
+      ));
+
+      minted = {
+        brand: await E(E(zoeService).getFeeIssuer()).getBrand(),
+        make: n => AmountMath.make(minted.brand, n),
+      };
+      psmTerms.anchorPerMinted = makeRatio(1n, anchor.brand, 1n, minted.brand);
+      psmTerms.governedParams.WantMintedFee = {
+        type: ParamTypes.RATIO,
+        value: makeRatio(1n, minted.brand, 1n),
+      };
+      psmTerms.governedParams.GiveMintedFee = {
+        type: ParamTypes.RATIO,
+        value: makeRatio(1n, minted.brand, 1n),
+      };
+      psmTerms.governedParams.MintLimit = {
+        type: ParamTypes.AMOUNT,
+        value: AmountMath.make(minted.brand, scale6(1_000_000)),
```

Comment:
slightly odd to not use `minted.make` defined a few lines up

Reply:
yup, missed refactoring